README.md

```diff
@@ -38,10 +38,20 @@ Makes Entur data available, so far only NeTEx route data.
    ENTUR_API_PATH          = timetable-admin/v1/timetable/download_netex_blocks/***
    ENTUR_ENV               = staging
 
+   CLEOS_ENV               = staging
```

Comment:
Not used

Reply:
Removed CLEOS_ENV from config/ragnarok_entur.php